MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeAppDelegateTests.m

```diff
@@ -0,0 +1,37 @@
+#import "MSTestFrameworks.h"
+#import "MSDistributePrivate.h"
+#import "MSDistributeAppDelegate.h"
+
+@interface MSDistributeAppDelegateTests : XCTestCase
+
+@end
+
+@implementation MSDistributeAppDelegateTests
+
+- (void)testOpenURLIsCalled {
+
+  // If.
+  MSDistributeAppDelegate *appDelegate = [[MSDistributeAppDelegate alloc] init];
+  id distributeMock = OCMPartialMock([MSDistribute sharedInstance]);
```

Comment:
All `OCMPartialMock` *should* `stopMocking` then to avoid impact to other tests

Reply:
> The real object can be returned to its original state by calling stopMocking. This is only necessary if the original state must be restored before the end of the test. The partial mock automatically calls stopMocking during its own deallocation.

From docs: http://ocmock.org/reference/#partial-mocks

Reply:
True for local objects, but I guess `sharedInstance` not deallocated here.

Reply:
distributeMock is local variable, so stopMocking method will be called after its deallocation.